cli/aws_orbit/remote_files/cdk/lambda_sources/cognito_post_authentication/k8s_manage.py

```diff
@@ -169,7 +169,7 @@ def manage_user_namespace(event: Dict[str, Any], api: client.CoreV1Api) -> None:
     all_ns = [
         item.get("metadata").get("name")
         for item in all_ns_raw["items"]
-        if item.get("metadata").get("name").endswith(user_name)
+        if item.get("metadata").get("name") and item.get("metadata").get("name").endswith(user_name)
```

Comment:
this should probably be:
```
if item.get("metadata", {}).get("name") and item.get("metadata", {}).get("name").endswith(user_name)
```
so on the off chance there's no metadata it doesn't blow up